gui/src/ocpn_frame.cpp

```diff
@@ -5063,7 +5063,7 @@ void MyFrame::HandleGPSWatchdogMsg(std::shared_ptr<const GPSWatchdogMsg> msg) {
     if (msg->wd_source == GPSWatchdogMsg::WDSource::position){
       bool last_bGPSValid = bGPSValid;
       bGPSValid = false;
-      m_fixtime = 0;    // Invalidate fix time
+      m_fixtime = -1;    // Invalidate fix time
```

Comment:
Changed from `0` to `-1` to make it consistent with https://github.com/OpenCPN/OpenCPN/blob/a3b30b9d764b4dc79caffca5115200aeac48a096/gui/src/ocpn_frame.cpp#L703